master/buildbot/reporters/utils.py

```diff
@@ -163,7 +163,7 @@ def getResponsibleUsersForBuild(master, buildid):
 
     # add owner from properties
     if 'owner' in properties:
-        blamelist.add(properties['owner'][0])
+        blamelist.update(properties['owner'][0])
```

Comment:
I guess this is rather a bug in generation of the owner property.

For me this property should be only one string, and not list of strings.